prereise/gather/data/BA_County_map.json

```diff
@@ -3006,7 +3006,7 @@
         "Washington__OR",
         "Yamhill__OR"
     ],
-    "ERCOT": [
+     "ERCOT": [
```

Comment:
We probably don't need this extra space here.

Reply:
In fact, if we have the ERCOT sub-zones, do we still need this `"ERCOT"` mapping at all?

Reply:
It is a duplication. The ERCOT section is removed. Some counties in ERCOT is assigned to ERCOT-N if they are not in any of the ERCOT subregions.  
'["Wheeler__TX", "Castro__TX", "Armstrong__TX", "Ochiltree__TX", "Swisher__TX", "Oldham__TX", "Carson__TX", "Roberts__TX", "Deaf Smith__TX", "Collingsworth__TX"]'